ui/job-view/details/tabs/TabsPanel.jsx

```diff
@@ -237,15 +237,19 @@ class TabsPanel extends React.Component {
           </TabPanel>
           <TabPanel>
             <SimilarJobsTab
-              repoName={repoName}
+              repoName={currentRepo.name}
               classificationMap={classificationMap}
               selectedJobFull={selectedJobFull}
             />
           </TabPanel>
           {showPerf && (
             <TabPanel>
               <PerformanceTab
-                repoName={repoName}
+                key={selectedJobFull.id}
+                selectedJobFull={selectedJobFull}
+                currentRepo={currentRepo}
```

Comment:
I noticed `PerformanceTab` doesn't really use `currentRepo`. Could this be skipped?

Reply:
It's needed for the `triggerGeckoProfileTask` task function, which is used in the `PerformanceTab`.